gravityflow/gravityflowformconnector
# Fix Entry Lookup appearing before Source Form is selected with Gravity Forms 2.5

## Description
Re: [Issue#13](https://github.com/gravityflow/backlog/issues/13)

## Testing instructions
1. Create an Update Fields step with Gravity Forms 2.5 installed.
2. The 'Entry lookup' will appear even before the Source Form is selected.
3. Now update to this branch and check again.
4. Entry lookup only appears after a Source Form is selected.
5. Repeat with Gravity Forms 2.4 to ensure nothing broke there.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes?
N/A

## Checklist:
